cms/djangoapps/contentstore/management/commands/tests/test_create_course.py

```diff
@@ -61,3 +61,35 @@ def test_all_stores_user_email(self, store):
         )
         # pylint: disable=protected-access
         self.assertEqual(store, modulestore()._get_modulestore_for_courselike(new_key).get_modulestore_type())
+
+    @ddt.data(ModuleStoreEnum.Type.split, ModuleStoreEnum.Type.mongo)
+    def test_get_course_with_different_case(self, default_store):
+        """
+        Tests that course can not be accessed with different case.
+
+        Scenario:
+            Create a course with lower case keys inside `bulk_operations` with `ignore_case=True`.
+            Verify that course is created.
+            Verify and that get course from store using same course id but different case is not accessible.
+        """
+        org = 'Org1'
+        number = 'Course1'
+        run = 'Run1'
+        with self.store.default_store(default_store):
+            lowercase_course_id = self.store.make_course_key(org.lower(), number.lower(), run.lower())
```

Comment:
@awaisdar001 This looks good, but I would suggest you can define the `org`, `number` and `run` in lower case and you don't need to use here `.lower()`.